### DISCUSSION TITLE
What's Renovate's default behavior for rollback PRs?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#9490] [Request Help] [answered]
I saw a issue where somebody had problems with the `rollbackPRs` property. I did not know we had this property at all, so I went looking in the docs.

To me the current docs are not clear on the default behavior.

The text seems to imply Renovate defaults to `true`: it makes a rollback PR when needed. If you don't want this behavior, you can turn this off by setting it to `false`.

The table says the default is `false`. But now the text makes no sense, why should I need to set it to `false` if the table says the default is `false` already?

So I think we're getting something mixed up here. 😄 🙈 
Once we're clear on what it should be, I'll make a PR to fix it. 😄 

https://docs.renovatebot.com/configuration-options/#rollbackprs

### rollbackPrs

Create PRs to roll back versions if the current version is not found in the registry.

| name | value |
| --- | --- |
| type | boolean |
| default | `false` |

Configure this to `false` either globally, per-language, or per-package if you want to disable Renovate's behavior of generating rollback PRs when it can't find the current version on the registry anymore.

**Comment by rarkins** [comment, marked as answer]
It's true for npm only, so the docs can be updated to reflect that.

**Reply by HonkingGoose** [reply]
Discussion will be resolved by PR #9494. 😄
